src/app/services/log.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
+import { LogLevel } from '../models/log-level.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LogService {
+  private readonly logLevel: LogLevel = new LogLevel();
```

Comment:
I think it'd make a lot more sense to make `LogLevel` an enum and export it from this file. I don't see any advantage of using a class here since it offers no extra functionality.

Also, I think it makes sense to just export the enum straight out of this class and allow consumers to import it from here. Anything that uses the `LogService` class will need to use the `LogLevel` enum so there isn't any advantage in putting the `LogLevel` enum in a more centralized location. 

Hopefully that makes sense!

Reply:
Absolutely! This and [another comment](https://github.com/kelseyroy/wangle/pull/5#discussion_r1291713240) about removing `logWith()` has me wondering whether it's worthwhile to keep the `LogLevel` at all? This could be over optimized and it could be as simple as:

```
export class LogService {
  info(msg: string)   { ... }
  error(msg: string) { ... }
  warn(msg: string)  { ... }
}
```